CHANGELOG.md

```diff
@@ -1,13 +1,28 @@
 # CHANGE LOG
 
-## 3.11.0-beta
+## 3.11.0
 
-### Bug Fixes
+### Updates
 - Support for Merch Studio and the Visual Merchandiser.
 - Upgraded the Algolia PHP client to version 3.3.2
 - Upgraded the Algolia insight 2.6.0
 - Preserve facet selections after adding an item to the cart from the PLP
 - Fixes related to Neural Search compatibility
+- Added the code to hide Pagination when the results have only one page in InstantSearch page
+- Increased the  character limit of Class filed in algoliasearch_queue table
```

Comment:
Increased the  character limit of class field in algoliasearch_queue, algoliasearch_queue_archive table